services/app-api/src/handlers/third_party_API_authorizer.ts

```diff
@@ -25,12 +25,32 @@ const jwtLib = newJWTLib({
     expirationDurationS: 90 * 24 * 60 * 60, // 90 days
 })
 
-const main: APIGatewayTokenAuthorizerHandler = async (
+const oauthJwtLib = newJWTLib({
+    issuer: 'mcreview-oauth',
+    signingKey: Buffer.from(jwtSecret, 'hex'),
+    expirationDurationS: 3600, // 1 hour
```

Comment:
@shamoons apologies if this was already discussed. This will change the expiration duration from 90 days to 1 hour. Is the ARMs team aware of this change?

Reply:
Done